src/Beberlei/Bundle/MetricsBundle/DependencyInjection/Configuration.php

```diff
@@ -18,21 +18,36 @@ public function getConfigTreeBuilder()
 
         $rootNode
             ->children()
-                ->scalarNode('default')->defaultValue('default')->end()
+                ->scalarNode('default')
+                    ->defaultNull()
+                ->end()
                 ->arrayNode('collectors')
                     ->isRequired()
                     ->useAttributeAsKey('name')
                     ->prototype('array')
                         ->children()
-                            ->scalarNode('type')->isRequired()->end()
-                            ->scalarNode('hostname')->end()
-                            ->scalarNode('server')->end()
-                            ->scalarNode('host')->end()
-                            ->scalarNode('port')->end()
-                            ->scalarNode('file')->end()
-                            ->scalarNode('username')->end()
-                            ->scalarNode('password')->end()
-                            ->scalarNode('connection')->end()
+                            ->scalarNode('type')->end()
+                            ->scalarNode('connection')->defaultValue('default')->end()
```

Comment:
this is wrong. The default Doctrine DBAL connection may have any name (and it is aliased as ``database_connection``)